modules/examples/routeguide/protocol/src/main/resources/application.conf

```diff
@@ -0,0 +1,14 @@
+rpc {
```

Comment:
This file wouldn't be part of the protocol, it might be more for the server side.